frontend/src/components/regions/UsersAndGroups/Presentation.js

```diff
@@ -50,17 +71,15 @@ const TeamSummaryStats = ({ summaryStats }) => {
         <TableRow>
           <TableCell>Pull Requests</TableCell>
           <TableCell>{summaryStats.totalOpenPrs}</TableCell>
-          <TableCell>
-            {oldestOpenPrTime ? oldestOpenPrTime.toLocaleString() : "-"}
+          <TableCell className={prStatusClassName}>
+          {getAgeString(openPrAge) ? getAgeString(openPrAge) : "-"}
```

Comment:
Rather:
```
          {openPrAge ? getAgeString(openPrAge) : "-"}
```

Why call getAgeString twice when you can call it once?  If there is no age then why would we want to check if we can make the age pretty before we make it pretty. Just make it pretty if it exists.